egs/librispeech/ASR/pruned_transducer_stateless3/onnx_check_all_in_one.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Xiaomi Corporation (Author: Yunus Emre Ozkose)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script checks that exported onnx models produce the same output
+with the given torchscript model for the same input.
+"""
+
+import os
+import argparse
+import logging
+
+import onnxruntime as ort
+import torch
+
+import onnx
+import onnxruntime
+import onnx_graphsurgeon as gs
```

Comment:
Could you add this dependency to
https://github.com/k2-fsa/icefall/blob/36eacaccb2f88e724045159a0f15bb20b82e46c4/requirements.txt#L8
and
https://github.com/k2-fsa/icefall/blob/36eacaccb2f88e724045159a0f15bb20b82e46c4/requirements-ci.txt#L25